### PR TITLE
Hibernate setup done

### PR DESCRIPTION
**Pull request by Madalin584**

### CODE REVIEW

File: src/main/java/sample/Main.java [hunk 1]
```diff
@@ -21,7 +27,21 @@ public void start(Stage primaryStage) throws IOException {
...
+    }
+
+    public static void createUser() {
+        User user = new User("test", "test1pass");
+        Transaction transaction = null;
+        try {
+            Session session = HibernateUtil.getSession();
+            session.save(user);
+            transaction = session.beginTransaction();
+            transaction.commit();
```
Comment (Madalin584) [inline src/main/java/sample/Main.java:42]: Test user was added to DB

File: src/main/java/sample/Main.java [hunk 2]
```diff
@@ -21,7 +27,21 @@ public void start(Stage primaryStage) throws IOException {
 
 
     public static void main(String[] args) {
-        launch(args);
-        System.out.println("Hello");
+        //launch(args);
+        //System.out.println("Hello");
```
Comment (stefan2212) [inline src/main/java/sample/Main.java:31]: delete uncomented code

File: src/main/java/sample/service/HibernateUtil.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        settings.put(Environment.DRIVER, "com.mysql.cj.jdbc.Driver");
+        settings.put(Environment.URL, "jdbc:mysql://localhost:3306/productsfrommarketplace_schema?useSSL=false");
+        settings.put(Environment.USER, "root");
+        settings.put(Environment.PASS, "root");
+        settings.put(Environment.DIALECT, "org.hibernate.dialect.MySQL5Dialect");
+        settings.put(Environment.SHOW_SQL, "true");
+        settings.put(Environment.CURRENT_SESSION_CONTEXT_CLASS, "thread");
+        settings.put(Environment.HBM2DDL_AUTO, "update");
+        configuration.setProperties(settings);
+        logger.info("A returnat configuratia");
```
Comment (stefan2212) [inline src/main/java/sample/service/HibernateUtil.java:58]: No Romanian code!! Only english

File: src/main/java/sample/model/User.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+@NoArgsConstructor
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Integer id;
+    @Column
+    private String username;
+    @Column
+    private String password;
+
```
Comment (stefan2212) [inline src/main/java/sample/model/User.java:23]: No role for user? How can I check if the user is employee or customer

**Pull request merged**